app-text/diffpdf/diffpdf-2.1.3-r3.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit desktop cmake
+
+DESCRIPTION="Program that textually or visually compares two PDF files"
+HOMEPAGE="https://web.archive.org/web/20250102202818/https://www.qtrac.eu/diffpdf-foss.html"
+SRC_URI="https://web.archive.org/web/20201229194512/http://www.qtrac.eu/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+BDEPEND="dev-qt/qttools[linguist]"
```

Comment:
needs :6 slot dep, but also, as you yourself require 6.8 minimum version in CMakeLists.txt, would have to define that minimum here too.